CoverMyDotNet.Tests/ClientTests.cs

```diff
@@ -204,6 +205,233 @@ public void Should_Client_Delete_Post_Correct_Data()
 				Assert.GreaterOrEqual(requestContent.IndexOf("Steve"), 0);
 			}	
 		}
-	}
-}
 
+		[Test]
+		public void Should_Client_Search_Drug_Use_Correct_Data()
+		{
+			string drug_name = Guid.NewGuid().ToString();
+			Dictionary<string, string> requestContent = new Dictionary<string, string>();
+			var requestHandlers = new List<MockHttpHandler>()
+			{
+				new MockHttpHandler("/drugs", "GET", (req, rsp, prm) => 
+				{
+					requestContent = prm;
+				})
+			};
+			using(new MockServer(DEFAULT_MOCK_PORT, requestHandlers))
+			{
+				_client.SearchDrugs(drug_name);
+				Assert.That(requestContent.Any(p => p.Key == "q"));
+				Assert.That(requestContent.Any(p => p.Value == drug_name));
+			}	
+		}
+
+		[Test]
+		public void Should_Client_Search_Drug_Parse_Data()
+		{
+			string drug_name = Guid.NewGuid().ToString();
+			Dictionary<string, string> requestContent = new Dictionary<string, string>();
+			var requestHandlers = new List<MockHttpHandler>()
+			{
+				new MockHttpHandler("/drugs", "GET", (req, rsp, prm) => 
+				{
+					return File.ReadAllText("Fixtures/Drugs.json");
```

Comment:
okay I added them in this commit https://github.com/covermymeds/CoverMyDotNet/pull/2/commits/ec41011a730e7d433965f6c168ef2ccb0711ca8e